src/style.css

```diff
@@ -0,0 +1,109 @@
+body {
+  color: #000;
+  margin: 0;
+  font-family: "Roboto", sans-serif;
+}
+
+.product-card {
+  background: #fff;
+  border: 1px solid #f3f3f3;
+  box-sizing: border-box;
+  border-radius: 5px;
+  padding: 30px 15px 15px;
+  width: 200px;
+}
+
+.image-wrap {
+  height: 134px;
+  margin: 0 auto;
+  padding-bottom: 44px;
+  width: 158px;
+}
+
+.image-wrap img {
+  width: 100%;
+  height: auto;
+}
+
+.product-name {
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 15px;
+  margin: 0;
+}
+
+.product-code {
+  font-size: 11px;
+  line-height: 13px;
+}
+
+.stars-list {
+  list-style: none;
+  padding-left: 0;
+  display: flex;
+  margin: 0;
+  width: 91px;
+  justify-content: space-between;
+}
+
+.rating {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 13px 0 19px;
+}
+
+.stars-item {
+  height: 15px;
+  width: 15px;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+.rating-text {
+  font-size: 11px;
+  line-height: 14px;
+  padding-top: 2px;
+}
+
+.price {
+  display: flex;
+  justify-content: space-between;
+  align-items: flex-end;
+  padding-bottom: 20px;
+}
+
+.price-title {
+  font-size: 11px;
+  line-height: 13px;
+}
+
+.cost {
+  font-weight: bold;
+  font-size: 16px;
+  line-height: 19px;
+  width: 79px;
```

Comment:
Don't use such kind of explicitly defined width. The card should be flexible. One item can have price `10` and another - `10000000`. The width should not be set.